Branch-SDK-Tests/BNCDeviceInfoTests.m

```diff
@@ -140,6 +140,8 @@ - (void)testV2Dictionary {
     XCTAssertNotNil([dict objectForKey:@"os"]);
     XCTAssertNotNil([dict objectForKey:@"sdk"]);
     XCTAssertNotNil([dict objectForKey:@"sdk_version"]);
+    
+    XCTAssertNil([dict objectForKey:@"disable_ad_network_callouts"]);
```

Comment:
Seems like this would all be more readable with more standard dictionary notation, e.g. `XCTAssertNil(dict[@"disable_ad_network_callouts"]);`. Just a general style/syntax comment. No need to change anything in this PR as far as I'm concerned.

Reply:
That's a good call.  I just kept going with the previous code, but really no need to.